README.md

```diff
@@ -1,11 +1,25 @@
 # HiddenLayer Model Scanner Github Action
 
-This action scans models stored in the repo or on S3. Results by default are stored in the Github Action Job Summary, with the ability to write the results to a Pull Request.
+Integrate model scanning into your continuous integration (CI) process with HiddenLayer's GitHub Actions (GHA) integration. This action can scan your models that are stored in a repository or on Amazon S3.
+
+By default, the results are stored in the GitHub Actions Job Summary. You can write the results to a GitHub Pull Request (see example below).
+
+The model scan results can display the following:
+
+- Green checkmark: No issues identified in the model.
+- Red X: Issues were identified in the model. See the scan results for more information.
 
 ![image](https://github.com/hiddenlayerai/hiddenlayer-model-scan-github-action/assets/9558507/dbebe8db-7e68-479d-a244-070238adf00d)
 
+For more information about GitHub Actions:
+
+- Read the [Understanding GitHub Actions](https://docs.github.com/en/actions/learn-github-actions/understanding-github-actions) page.
+- Read the [GitHub Actions Quickstart](https://docs.github.com/en/actions/quickstart) for a quick overview.
+
 ## Inputs
 
+`id` (required): Scans a model file or scans all model files in a folder. Supported values are: `scan_model` and `scan_model_folder`.
```

Comment:
This field is a Github Action field and can be anything the user wants. It is the name of the task in the overall GitHub workflow.

Reply:
OK. Then I'll remove it. I thought the user needed to select one or the other.